### ISSUE TITLE
[BUG] Some releases are not published on pkg.go.dev

### ISSUE THREAD

**Issue by antoninferrand** [stackrox/kube-linter#599]
**Describe the bug**
Some releases of kube-linter are not published : https://pkg.go.dev/golang.stackrox.io/kube-linter?tab=versions ([0.5.0](https://github.com/stackrox/kube-linter/releases/tag/0.5.0), [0.5.1](https://github.com/stackrox/kube-linter/releases/tag/0.5.1) and [0.6.0](https://github.com/stackrox/kube-linter/releases/tag/0.6.0) are missing).

**To Reproduce**
Run `go get github.com/stackrox/kube-linter@v0.5.0`

**Sample YAML input**
//

**Expected behavior**
There should not be gaps in the available releases.

**Screenshots**
<img width="381" alt="image" src="https://github.com/stackrox/kube-linter/assets/131135452/bc0b73c0-b938-4513-b4de-2e5c7ff132f9">

**Comment by janisz** [comment]
Good point. We did not add semantic tags to releases priori to v.6.1 this was fixed with https://github.com/stackrox/kube-linter/pull/497 but we should add `v` prefixed tags to old releases as well. Double tagging should not break anything
